esmvalcore/experimental/templates/RecipeInfo.j2

```diff
@@ -0,0 +1,37 @@
+<h1>{{ info.name }}</h1>
```

Comment:
Would it make sense for these templates to be in a folder called 'output' or something like that?

Reply:
I think 'templates' is a suitable name. Jinja2 works with 'environments', so once an environment path is defined, you can retrieve templates everywhere.

Reply:
Yeah I agree templates makes sense, they're often used in web projects I think. But since ESMValTool is not really a web project I'm not sure if it is immediately obvious what kind of templates they are in this case. That's why I thought it might make sense to make this a bit more explicit.